### PR TITLE
Zwei Association + Hardhead Office ERTs!

### PR DESCRIPTION
**Pull request by John-Dang**
## About The Pull Request
Adds the Zwei Association ERT! Did you already use up your Rabbit call for the day with this Lobotomy Corporation branch office? That's okay, this is the first of a variety of offices and syndicates you can call for help, for less energy perhaps? They're mostly focused on red damage and deathball tactics, with decent protective coats and neat uniforms. We also renamed and redescribed the ordeal meat and fixed up a few other descriptions. Also added Hardhead Office fixer squad, for dealing black damage.

## Why It's Good For The Game
Variety among the ERTs is good, its fun, keeps things from feeling tired! Horde of interns would be proud.

## Changelog
:cl:
add: Added the Zwei Association ERT for admins to choose. Also, Hardhead Office, the black damage squad to kill robots.
tweak: tweaked a few items that I gave the Zwei Association
balance: the zwei sword does 40 red damage and the zwei association coat has a general protection of III. Hardhead weapon does 30 black damage, and the hardhead uniform has general protection of III. Hardhead leader has a 40 black damage weapon.
/:cl:

### CODE REVIEW

File: code/datums/ert.dm [hunk 1]
```diff
@@ -71,20 +71,29 @@
...
-	leader_role = /datum/antagonist/ert/intern/leader
+/datum/ert/zwei
+	roles = list(/datum/antagonist/ert/zwei)
+	leader_role = /datum/antagonist/ert/zwei/captain
 	teamsize = 7
 	opendoors = FALSE
-	rename_team = "Horde of Interns"
-	mission = "Assist in conflict resolution."
-	polldesc = "an unpaid internship opportunity with Nanotrasen"
+	rename_team = "Zwei Association Raid"
```
Comment (EgorDinamit) [inline code/datums/ert.dm:79]: "Raid" sounds quite hostile.

File: code/datums/ert.dm [hunk 2]
```diff
@@ -71,20 +71,29 @@
...
+	mission = "Assist in conflict resolution for Lobotomy Corporation."
+	polldesc = "an opportunity for fixers to make a name for themselves, or die"
```
Comment (EgorDinamit) [inline code/datums/ert.dm:81]: This shows up for ghosts as "Do you want to join as" message, should probably be a proper description of the ERT.

File: code/game/objects/items/cards_ids.dm [hunk 3]
```diff
@@ -512,9 +512,9 @@ update_label()
 		..()
 
 /obj/item/card/id/centcom
```
Comment (EgorDinamit) [inline code/game/objects/items/cards_ids.dm:514]: You should add a new type for them instead.

File: code/game/objects/items/devices/radio/headset.dm [hunk 4]
```diff
@@ -223,8 +223,8 @@ GLOBAL_LIST_INIT(channel_tokens, list(
 	AddComponent(/datum/component/wearertargeting/earprotection, list(ITEM_SLOT_EARS))
 
 /obj/item/radio/headset/headset_cent
```
Comment (EgorDinamit) [inline code/game/objects/items/devices/radio/headset.dm:225]: Same here, don't rename headset used by Rabbits, make a new one.

File: code/game/objects/items/weaponry.dm [hunk 5]
```diff
@@ -63,6 +63,8 @@ for further reading, please see: https://github.com/tgstation/tgstation/pull/301
 	flags_1 = CONDUCT_1
 	slot_flags = ITEM_SLOT_BELT | ITEM_SLOT_BACK
 	force = 40
+	damtype = RED_DAMAGE
```
Comment (EgorDinamit) [inline code/game/objects/items/weaponry.dm:66]: All items, by default, are red damage. This is not needed.

Comment (John-Dang) [reply code/game/objects/items/weaponry.dm]: Specification removed from weapon.

File: code/game/objects/items/devices/radio/headset.dm (hunk 4 above)
Comment (John-Dang) [reply code/game/objects/items/devices/radio/headset.dm]: New headset made for them.

File: code/game/objects/items/cards_ids.dm (hunk 3 above)
Comment (John-Dang) [reply code/game/objects/items/cards_ids.dm:514]: New ID cards are now made.

File: code/datums/ert.dm (hunk 2 above)
Comment (John-Dang) [reply code/datums/ert.dm]: Changed the polldescs for the ERT groups.

File: code/datums/ert.dm (hunk 1 above)
Comment (John-Dang) [reply code/datums/ert.dm]: Changed "raid" to "contract".

**Pull request closed without merging**